server/setup_postgresql.sh

```diff
@@ -1,10 +1,35 @@
 #!/bin/bash
 
-pg_password=$1
-pg_user=$2
-pg_host=$3
-pg_dbname=$4
-pg_port=$5
+
```

Comment:
Just curious... why did this script need to change?

Reply:
To run the migrations I am now actually firing up a container on the cluster and executing this script from within it. All the DB connection information is injected as environment variables so this modification was the allow the script to read them from the environment if they are there. If they are not there, it should then expect them as input arguments as before so should continue to function in the old manner as well if required.